docs/en/03-benchmark/benchmark.md

```diff
@@ -1061,6 +1090,35 @@ Users can directly test the performance through [how_to_evaluate_a_model.md](../
     <td align="center">0.8497</td>
     <td align="center">0.8469</td>
     <td align="center"> - </td>
+    <td align="center">-</td>
+    <td align="center">-</td>
+  </tr>
+    <tr>
+    <td align="center" rowspan="2"><a href="https://github.com/open-mmlab/mmediting/blob/master/configs/restorers/edsr/edsr_x2c64b16_g1_300k_div2k.py">EDSRx2</a></td>
+    <td align="center" rowspan="2">Super Resolution</td>
+    <td align="center" rowspan="2">Set5</td>
+    <td align="center">PSNR</td>
+    <td align="center">-</td>
```

Comment:
Could you fill the metric values of PyTorch column?

Reply:
sure, I filled the metric values of PyTorch column just now